libs/mat-table-filter/src/lib/mat-table-filter-button/mat-table-filter-button.component.ts

```diff
@@ -29,6 +29,7 @@ export class MatTableFilterButtonComponent
 {
   public intlService = inject(MatTableFilterIntlService);
   private cd = inject(ChangeDetectorRef);
+  private isCaseSensitive = true;
```

Comment:
I think there is no need to have another field to hold case sensitivity state. You can just use `sensitivityType`